src/components/views/UserPage.js

```diff
@@ -159,19 +123,19 @@ const UserPage = () =>{
         </div>
     )
 
-    if(preferences){
+    if(user){
         userPreferences = (
             <div>
                 <div className="userPage title">Preferences</div>
                 <table className="userPage table">
                     <tbody>
                     <tr className="userPage player-info-container">
                         <td> Age Preference:</td>
-                        <td className="userPage td"> {preferences.minAge} Years - {preferences.maxAge} Years</td>
+                        <td className="userPage td"> {user.minAge} Years - {user.maxAge} Years</td>
                     </tr>
                     <tr className="userPage player-info-container">
                         <td> Gender Preferences: </td>
```

Comment:
you have the word 'preference' in the heading, so I would remove it from here and just call it gender.

Reply:
done, same for age aswell